src/pages/glossary/glossary.scss

```diff
@@ -10,51 +10,47 @@
     overflow-wrap: break-word;
     color: black;
 }
-.glossary-term-container {
-    padding: 1em 1em;
-    @media (min-width: 767px) and (max-width: 1023px) {
-        padding: .5em;
-    }
-    @media (max-width: 766px) {
-        padding: .5em;
-    }
-}
+
 #glossary-term-wrapper {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-around;
```

Comment:
Nothing that you changed, but as I am looking at this, the term buttons look much better with `justify-content: space-between;`. Can you update it to that? Thanks!